pkgs/applications/office/ledger-autosync/default.nix

```diff
@@ -46,8 +51,11 @@ python3Packages.buildPythonApplication rec {
   # Checks require ledger as a python package,
   # ledger does not support python3 while ledger-autosync requires it.
   checkInputs = with python3Packages; [ ledger hledger nose mock ];
+
+  # Disable some non-passing tests:
+  # https://github.com/egh/ledger-autosync/issues/127
   checkPhase = ''
-    nosetests -a generic -a ledger -a hledger
+    pytest -k 'not test_payee_match and not test_args_only'
   '';
```

Comment:
Use pytestCheckHook instead.

Reply:
Much nicer, thank you for your suggestion.